packages/hosted-assets/index.ts

```diff
@@ -8,6 +8,8 @@ const ORIGIN_BASE_URL = "https://d1e7r7b0lb8p4d.cloudfront.net"
  * assetUrl("some/blob.png") -> "https://<origin>/some/blob.png"
  *
  * @see https://github.com/cultureamp/kaizen-design-system-assets/
+ *
+ * @deprecated import from `@kaizen/components` instead
```

Comment:
For external consumers? Does it come through from `@kaizen/components` or from a subpath?

Reply:
Comes through `@kaizen/components`